src/commands/format.ts

```diff
@@ -0,0 +1,29 @@
+import { CommandDef } from './command-def';
+import { Message } from 'discord.js';
+import { addFormatting } from './add-formatting';
+
+export const format: CommandDef = {
+  prefix: 'format',
+  description: 'Formats the code contained in the message ID.',
```

Comment:
Where and how is the user suppose to get the messageId? My original understanding was that this function would load via a discord URL?

Reply:
So...
That was the original plan.
But discord.js doesn't have a way to find a message by the url, that I could see.
So I went with this route instead.
But just now I've realised how I could parse the ID *from* the URL, and I'll go ahead and try to factor that into the command instead. Allow me a few minutes and I'll have this changed. 

Also, the ID can be copied just like the url (right-click drop down) if you've got dev mode turned on 🙂

Reply:
This is changed now.